CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 2.8.8)
+cmake_minimum_required (VERSION 3.0)
```

Comment:
Just saw that we already announced in 2016-04-29_0.8.16.md

> This might be the last release supporting wheezy, because
it gets more and more time-intensive to find workarounds
for the old compiler. The C++11 regex do not work at all.

So actually you are right, we should not worry and finally drop the support (and pump the cmake version number).